modules/js/src/embindgen.py

```diff
@@ -155,9 +103,14 @@ def makeWhiteList(module_list):
                 wl[k] = m[k]
     return wl
 
-white_list = makeWhiteList([core, imgproc, objdetect, video, dnn, features2d, calib3d])
+track = ""
+if os.environ["OPENCV_JS_WHITELIST"] == "opencv_js.config.py":
+    track = os.path.join(os.path.dirname(os.path.abspath(__file__)), os.environ["OPENCV_JS_WHITELIST"])
```

Comment:
Let's move `opencv_js.config.py` near to `build_js.py` and create a full path inside the `build_js.py`